test/Microsoft.AspNetCore.Server.Kestrel.Performance/RequestParsing.cs

```diff
@@ -23,6 +23,11 @@ public class RequestParsing
 
         private const string plaintextRequest = "GET /plaintext HTTP/1.1\r\nHost: www.example.com\r\n\r\n";
 
+        private const string plaintextTechEmpower = "GET /plaintext HTTP/1.1\r\n" +
```

Comment:
Let's just replace plaintextRequest with this. I don't want to add too many benchmark methods to RequestParsing, and this is very similar to plaintextRequest just slightly more realistic. This also get's us a pipelined version of this test without adding even more runs.

@cesarbs said he would make this change.